swatch-tally/deploy/clowdapp.yaml

```diff
@@ -831,6 +855,88 @@ objects:
             - name: logs
               emptyDir:
 
+      - name: opt-in-org-id-populator
+        podSpec:
+          image: ${ORG_ID_POPULATOR_IMAGE}:${ORG_ID_POPULATOR_IMAGE_TAG}
+          command:
+            - ./org-id-column-populator
+            - -H
+            - $(DATABASE_HOST)
+            - -p
+            - $(DATABASE_PORT)
+            - -u
+            - $(DATABASE_USERNAME)
+            - -w
+            - $(DATABASE_PASSWORD)
+            - -n
+            - $(DATABASE_DATABASE)
+            - -a
+            - account_number
+            - -o
+            - org_id
+            - -t
+            - account_config
+            - -b
+            - $(ORG_ID_POPULATOR_BATCH_SIZE)
+            - --ean-translator-addr
+            - http://${TENANT_TRANSLATOR_HOST}:${TENANT_TRANSLATOR_PORT}
+            - -s
+            - disable
```

Comment:
@mstead not 100% sure on this but I think we'll have to parameterize the ssl -s argument. The default for stage and prod is 'verify-full' and 'disable' is only EE.